tools/hsettp/hsettp.c

```diff
@@ -1607,7 +1640,7 @@ root_usage(FILE * const output)
     print_operations(output);
 
     if (verbosity == 0)
-        fprintf(output, "\nUse -hv to show all operations\n\n");
+        fprintf(output, "\nUse -hv to show all operations\n");
```

Comment:
Suggest `Use -hv to show additional infrequently used operations`

Reply:
Makes more sense